pom.xml

```diff
@@ -67,16 +67,16 @@
     <httpcore-version>4.4.16</httpcore-version>
     <insight-version>1.2.0.Beta4</insight-version>
     <jackson-version>2.15.2</jackson-version>
-    <jakarta-jms-api-version>2.0.3</jakarta-jms-api-version>
+    <jakarta-jms-api-version>3.1.0</jakarta-jms-api-version>
```

Comment:
This property seems to be a duplicate of `jakarta-jms-api-v3-version`, one of them should be removed.

Reply:
Fixed as part of activemq-client-jakarta changes, that include adding Maven relocation info for that module.